src/AutoRest.CSharp/Output/Models/RestClientHelpers.cs

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using AutoRest.CSharp.Generation.Types;
+using AutoRest.CSharp.Input;
+using AutoRest.CSharp.Output.Builders;
+using AutoRest.CSharp.Output.Models.Requests;
+using AutoRest.CSharp.Output.Models.Responses;
+using AutoRest.CSharp.Output.Models.Serialization;
+using AutoRest.CSharp.Output.Models.Shared;
+using AutoRest.CSharp.Output.Models.Types;
+using AutoRest.CSharp.Utilities;
+using Azure.Core;
+using Request = AutoRest.CSharp.Output.Models.Requests.Request;
+using StatusCodes = AutoRest.CSharp.Output.Models.Responses.StatusCodes;
+
+namespace AutoRest.CSharp.Output.Models
+{
+    internal static class RestClientHelpers
```

Comment:
Is there a reason to have builder and helpers both?

Reply:
`RestClientBuilder` is a generic class you instance to generate the common code between DataPlaneRestClient and LowLevelRestClient.

`RestClientHelpers` is a bag of random helper static methods.

I separated them just because of that difference, instance vs static. I can combine if you disapprove.